src/providers/bttv/BttvEmotes.cpp

```diff
@@ -94,7 +94,7 @@ namespace {
                     ImageSet{
                         Image::fromUrl(getEmoteLinkV3(id, "1x"), 1),
                         Image::fromUrl(getEmoteLinkV3(id, "2x"), 0.5),
-                        Image::fromUrl(getEmoteLinkV3(id, "3x"), 0.25),
+                        Image::fromUrl(getEmoteLinkV3(id, "3x"), 0.286),
```

Comment:
I don't think `0.286` is the right value (at least not for all emotes). For `catJAM` it's `28/112 = 0.25` and `KKona` it's `25/100 = 0.25`.